### PR TITLE
Refinery: Add rule regarding Pull Request title prefix / Remove code …

### PR DESCRIPTION
**Pull request by mjansenDatabay**
…style paragraph / Fix language issues

### CODE REVIEW

File: src/Refinery/CONTRIBUTING.md [hunk 1]
```diff
@@ -46,16 +45,12 @@ announced one month in advance on the [Jour Fixe](https://docu.ilias.de/goto_doc
...
+* The title of the Pull Request should be prefixed with `Refinery:`.
 * The Pull Request must pass the CI integration.
   Be aware of the currently supported PHP versions and optimize your code according
   to the supported versions.
 * Keep the Pull Request as small as possible.
   Avoid unnecessary changes to speed up the review process.
 * Use readable and understandable commit messages, so the reviewer can understand the
   intention of each commit.
-
-## Code Style
```
Comment (mjansenDatabay) [inline src/Refinery/CONTRIBUTING.md:58]: I removed this because ILIAS has a general coding style guideline.

File: src/Refinery/CONTRIBUTING.md [hunk 2]
```diff
@@ -16,12 +16,11 @@ about the function of this role.
 
 ## Reporting Bugs
 
-Bugs should be reported via https://mantis.ilias.de and
-one of the
-[coordinators](/src/Refinery/maintenance.json)
-must be added to the bug ticket.
+Bugs must be reported via https://mantis.ilias.de in the `Refinery`
+category, One of the [coordinators](/src/Refinery/maintenance.json)
```
Comment (legionth) [inline src/Refinery/CONTRIBUTING.md]: If you want to begin a new sentence, we should  replace `,` with `.`.

**Review by legionth — CHANGES_REQUESTED** [review]
Just a little typo here.

Otherwise LGTM :+1:

**Review by legionth — APPROVED** [review]
LGTM :+1:

**Review by klees — APPROVED** [review]
Thx!

**Pull request merged**